packages/circuits/utils/regex.circom

```diff
@@ -26,6 +27,10 @@ template SelectRegexReveal(maxArrayLen, maxRevealLen) {
     signal isPreviousZero[maxArrayLen];
     signal isAboveMaxRevealLen[maxArrayLen];
 
+    // Assert startIndex < maxArrayLen
```

Comment:
@SoraSuegami Isn't this check already enforced in VarShiftLeft - https://github.com/zkemail/zk-email-verify/blob/main/packages/circuits/utils/array.circom#L121-L122 ?

Reply:
@SoraSuegami  ^ Ignore above comment. We need both (bit range check and less thank check).